src/components/value-list-item/value-list-item.tsx

```diff
@@ -181,6 +184,12 @@ export class ValueListItem
    */
   @Event({ cancelable: true }) calciteListItemRemove: EventEmitter<void>; // wrapped pick-list-item emits this
 
+  /**
+   * @internal
+   */
+  @Event({ cancelable: true })
+  calciteValueListItemDragHandleBlur: EventEmitter<ListItemAndHandle>;
```

Comment:
Is it possible to use the `getHandleAndItemElement` util to avoid passing the item and handle here?

Reply:
i tried doing so but the `focusEvent`   doesn't show up the handle element `composedPath` or `event.target`